### PR TITLE
Fr11.1 help

### PR DESCRIPTION
**Pull request by andehbrown**

### CODE REVIEW

**Comment by Rubberduck1** [comment]
@andehbrown:  If workaround 1 does not work, try workaround 2.

Workaround 1: Switch to the main branch at the bottom right and then enter the command “git pull” in the terminal to update everything.
Screenshot 2022-03-12 at 05 36 52

Workaround 2: Remove the project from your local directory. Then restart IntelliJ and select the option “Get from VSC”. Then copy the link from the GitHub repository and choose a local location. The project should then open clean and up-to-date.
Screenshot 2022-03-12 at 05 50 52

File: newbank/server/NewBank.java [hunk 1]
```diff
@@ -143,6 +143,9 @@ public synchronized String processRequest(CustomerID customer, String request) {
                         return acceptLoan(customer, Integer.parseInt(stringInputs[1]), stringInputs[2]);
                     }
                 }
+                case "HELP": {
```
Comment (rej696) [inline newbank/server/NewBank.java:146]: Good

File: newbank/server/NewBank.java [hunk 2]
```diff
@@ -311,4 +314,15 @@ private String generateAccountNumber() {
         return numberAsString;
     }
 
+    private String getHelp(){
```
Comment (rej696) [inline newbank/server/NewBank.java:317]: Good

**Review by rej696 — APPROVED** [review]
I think it's important that we allow these as command line arguments. It's a standard behaviour for command line applications, and the first thing a potential user would do to understand how to use the program. Potentially the system could display a message saying "run the program and enter the HELP command" rather than displaying the full help page. This can be handled on a separate pull request

**Pull request merged**